From d7723b630e42c0a2bf36dcc6f01413573f743398 Mon Sep 17 00:00:00 2001
From: Yaroslav Bolyukin <iam@lach.pw>
Date: Tue, 6 Feb 2024 19:53:22 +0100
Subject: [PATCH] fix: submodules are fetched

---
 setup.py | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/setup.py b/setup.py
index b49a677..afc86e1 100644
--- a/setup.py
+++ b/setup.py
@@ -136,7 +136,7 @@ def build_for_cuda():
         cc_flag.append("-gencode")
         cc_flag.append("arch=compute_90,code=sm_90")
 
-    subprocess.run(["git", "submodule", "update", "--init", "csrc/cutlass"], check=True)
+    # subprocess.run(["git", "submodule", "update", "--init", "csrc/cutlass"], check=True)
     ext_modules.append(
         CUDAExtension(
             name="flash_attn_2_cuda",
@@ -238,16 +238,16 @@ def build_for_rocm():
 
     rename_cpp_to_hip(fa_sources)
 
-    subprocess.run(
-        [
-            "git",
-            "submodule",
-            "update",
-            "--init",
-            "csrc/flash_attn_rocm/composable_kernel",
-        ],
-        check=True,
-    )
+    # subprocess.run(
+    #     [
+    #         "git",
+    #         "submodule",
+    #         "update",
+    #         "--init",
+    #         "csrc/flash_attn_rocm/composable_kernel",
+    #     ],
+    #     check=True,
+    # )
     ext_modules.append(
         CUDAExtension(
             name="flash_attn_2_cuda",
-- 
2.43.0

