From 2f785e42293d8400245a07da0ea0873b476eed33 Mon Sep 17 00:00:00 2001
From: Yaroslav Bolyukin <iam@lach.pw>
Date: Fri, 8 Mar 2024 14:37:20 +0100
Subject: [PATCH] fix: remove submodule fetch

---
 setup.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 75c92cb..ce35255 100644
--- a/setup.py
+++ b/setup.py
@@ -91,7 +91,7 @@ ext_modules = []
 
 # We want this even if SKIP_CUDA_BUILD because when we run python setup.py sdist we want the .hpp
 # files included in the source distribution, in case the user compiles from source.
-subprocess.run(["git", "submodule", "update", "--init", "csrc/cutlass"])
+# subprocess.run(["git", "submodule", "update", "--init", "csrc/cutlass"])
 
 if not SKIP_CUDA_BUILD:
     print("\n\ntorch.__version__  = {}\n\n".format(torch.__version__))
@@ -344,4 +344,4 @@ setup(
     setup_requires=[
         "psutil"
     ],
-)
\ No newline at end of file
+)
-- 
2.43.1

